src/main/java/devs/toz/todolistmicroservice/ToDoListMicroserviceApplication.java

```diff
@@ -16,6 +16,6 @@ public static void main(String[] args) {
     @GetMapping("/")
```

Comment:
I would like for you to create a controller package where you put a class named something like "TestController" and move the method there.